Re-factoring all of our code to meet MVC standards

Alex-Stan-1 opened the pull request:
Created three new classes: GameController, GameView, GameModel.

These classes make up the content from Game.cs (which is now removed) and also the Program.cs file (which still remains, but has been gutted).

Also added header comments to all new files.

Alex-Stan-1 committed:
Re-factoring all of our code to meet MVC standards

Alex-Stan-1 committed:
oops, forgot to add GameView

Skypho21 reviewed (dismissed):
This refactor is awesome! It's well documented and easy to follow. Thanks for putting in such good work!

voltaicturtle committed:
Update GameModel.cs

adjusted to meet the "new" json format

voltaicturtle committed:
Update GameModel.cs

Changed an inconsistency I noticed between the save and load functions. Could have caused a bug but probably not, its good to change anyways.

voltaicturtle committed:
Update GameModel.cs

oops didnt do it right the first time

noahirg reviewed (commented):
This looks amazing. You did a really good job. Thank you.

noahirg reviewed (approved):
Looks good
